eng/azure-sdk-deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <PublishDepsFilePath>$(IntermediateOutputPath)\azure-sdk-deps.json</PublishDepsFilePath>
+    <SdkRootPath>..\sdk</SdkRootPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExcludeProjects Include="$(SdkRootPath)\**\*Test*.csproj" />
```

Comment:
Should samples also be excluded?

Reply:
There shouldn't be any samples under the src folder so I think we are good. I only needed to add the Test exclude to filter out the TestFramework library which is structured like a shipping library.